### ISSUE TITLE
initial setup fails - no dir

### ISSUE THREAD

**Issue by crishaddad** [Crocmagnon/cleantoots#9]
Initial setup fails because the config dir hasn't been created yet:

```bash
$ ./cleantoots config setup 
[output truncated]  
ValueError: Unable to configure handler 'file': [Errno 2] No such file or directory: '/home/username/.config/cleantoots/cleantoots.log'
$ mkdir -p /home/username/.config/cleantoots/
$ ./cleantoots config setup 
/home/username/.config/cleantoots/config.ini written.
[output truncated] 
```

**Comment by Crocmagnon** [comment]
Thanks for the heads up! I’ll take a look, probably tomorrow.
It’s about the logging module which isn’t even used at that time.

**Comment by Crocmagnon** [comment]
@crishaddad this is fixed in version 0.3.6 available on PyPI, thanks again for the report! 🌟
